src/registry/textarea/edit.tsx

```diff
@@ -0,0 +1,245 @@
+            'autocomplete',
+            'disabled',
+            'placeholder',
+            'showCharCount',
+            'autoExpand',
+            'rows'
+          )}
+        />
+        <BuilderTabs.Advanced hasErrors={hasAnyError('conditional')} />
+        <BuilderTabs.Validation hasErrors={hasAnyError('validate')} />
+        <BuilderTabs.Translations hasErrors={hasAnyError('openForms.translations')} />
+      </TabList>
+
+      {/* Basic tab */}
+      <TabPanel>
+        <Label />
+        <Key isManuallySetRef={isKeyManuallySetRef} generatedValue={generatedKey} />
+        <Description />
+        <Tooltip />
+        <PresentationConfig />
+        <Multiple<TextareaComponentSchema> />
+        <Hidden />
+        <ClearOnHide />
+        <IsSensitiveData />
+        <DefaultValue multiple={!!values.multiple} rows={values.rows} />
+        <AutoComplete />
+        <ReadOnly />
+        <Placeholder />
+        <ShowCharCount />
+        <AutoExpand />
+        <NumberOfRows />
+      </TabPanel>
+
+      {/* Advanced tab */}
+      <TabPanel>
+        <SimpleConditional />
+      </TabPanel>
+
+      {/* Validation tab */}
+      <TabPanel>
+        <Validate.Required />
+        <Validate.ValidatorPluginSelect />
+        <Validate.MaxLength />
+        <Validate.RegexValidation />
+        <Validate.ValidationErrorTranslations />
+      </TabPanel>
+
+      {/* Translations */}
+      <TabPanel>
+        <Translations.ComponentTranslations<TextareaComponentSchema>
+          propertyLabels={{
+            label: intl.formatMessage(LABELS.label),
+            description: intl.formatMessage(LABELS.description),
+            tooltip: intl.formatMessage(LABELS.tooltip),
+            defaultValue: intl.formatMessage(LABELS.defaultValue),
+            placeholder: intl.formatMessage(LABELS.placeholder),
+          }}
+        />
+      </TabPanel>
+    </Tabs>
+  );
+};
+
+/*
+  Making this introspected or declarative doesn't seem advisable, as React is calling
+  React.Children and related API's legacy API - this may get removed in future
+  versions.
+
+  Explicitly specifying the schema and default values is therefore probbaly best, at
+  the cost of some repetition.
+ */
+EditForm.defaultValues = {
+  // basic tab
+  label: '',
+  key: '',
+  description: '',
+  tooltip: '',
+  showInSummary: true,
+  showInEmail: false,
+  showInPDF: true,
+  multiple: false,
+  hidden: false,
+  clearOnHide: true,
+  isSensitiveData: false,
+  defaultValue: '',
+  autocomplete: '',
+  disabled: false,
+  placeholder: '',
+  autoExpand: false,
+  showCharCount: false,
+  rows: 3,
+  // Advanced tab
+  conditional: {
+    show: undefined,
+    when: '',
+    eq: '',
+  },
+  // Validation tab
+  validate: {
+    required: false,
+    plugins: [],
+    maxLength: 10000,
+    pattern: '',
+  },
+  translatedErrors: {},
+  registration: {
+    attribute: '',
+  },
+};
+
+interface DefaultValueProps {
+  multiple: boolean;
+  rows: number;
```

Comment:
I think this can be ignored and we can pin it to 3 rows in the builder edit form.